corehq/apps/export/utils.py

```diff
@@ -79,6 +79,7 @@ def _strip_repeat_index(index):
 
 
 def _convert_index_to_path(index):
+    from corehq.apps.export.models.new import MAIN_TABLE
     if index == '#':
         return MAIN_TABLE
     elif _is_repeat(index):
```

Comment:
nice this implementation turned out well. damn you github and removing commit level comments